dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/instrumentation/decorator/HttpServerDecorator.java

```diff
@@ -28,7 +28,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class HttpServerDecorator<REQUEST, CONNECTION, RESPONSE, CARRIER>
+public abstract class HttpServerDecorator<REQUEST, CONNECTION, RESPONSE, REQUEST_CARRIER>
```

Comment:
Does it make sense to correct names? I feels a bit confused, because we have `REQUEST` and `RESPONSE` but they are unrelated to each other?
```suggestion
public abstract class HttpServerDecorator<REQUEST, CONNECTION, RESPONSE_CARRIER, REQUEST_CARRIER>
```

Reply:
That is not correct, since `REQUEST, CONNECTION, RESPONSE` are what the are named, but the `CARRIER` is the type that has the http **_request_** headers on it, which sometimes is different from the `REQUEST`. That is why I tried to make it clearer by naming it `REQUEST_CARRIER`.